src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
This line disables ESLint control-has-associated-label rule. Make sure this rule is disabled for a valid reason. It's generally recommended to follow accessibility standards.

Reply:
Consider following the previous recommendation